File: Server/BusinessLayer/Models/FlightBookInfo.cs
using System;
using Common;

namespace BusinessLayer.Models
{
    public class FlightBookInfo
    {
        public int? Id { get; set; }
        public int FlightId { get; set; }
        public int SuitcaseOverloadCost { get; set; }
        public int HandLuggageOverloadCost { get; set; }
        public BookType BookType { get; set; }
        public DateTimeOffset BookTime { get; set; }
        public int AccountId { get; set; }
        public SeatBook[] SeatBooks { get; set; }
    }
}

File: Frontend/src/services/flight-models/flight-book-info.js
export default class FlightBookInfo {
    flightId;
    suitcaseOverloadCost;
    handLuggageOverloadCost;
    seatBooks;

    constructor(flightId, suitcaseOverloadCost, handLuggageOverloadCost, seatBooks) {
        this.flightId = flightId;
        this.suitcaseOverloadCost = suitcaseOverloadCost;
        this.handLuggageOverloadCost = handLuggageOverloadCost;
        this.seatBooks = seatBooks;
    }
}